pyproject.toml

```diff
@@ -1,2 +1,89 @@
 [tool.black]
-target_version = ['py37']
\ No newline at end of file
+target_version = ['py38']
+
+[tool.poetry]
+name = "eq-questionnaire-runner"
+version = "0.1.0"
+description = "Python-Flask server side web application"
+authors = ["ONSDigital"]
+
+[tool.poetry.dev-dependencies]
```

Comment:
Defining dev dependencies like this is deprecated. You need to use groups https://python-poetry.org/docs/master/managing-dependencies/. So here we should use `[tool.poetry.group.dev.dependencies]`

Reply:
Changed to `[tool.poetry.group.dev.dependencies]`